src/api/java/com/minecolonies/api/compatibility/CompatibilityManager.java

```diff
@@ -801,6 +791,45 @@ private void discoverEnchantments()
         }
     }
 
+    /**
+     * Calculate the crusher modes from the config file.
+     */
+    private void discoverCrusherModes()
```

Comment:
Somethign went wrong in this class

Reply:
Weird. I didn't touch that

Reply:
OH I FIGURED IT OUT

You guys did something major with this class recently, so this PR is now recommending, essentially, reversion to the previous file. I'll try to update that bit later today so it's not recommending complete reversion